indicators.tcl

```diff
@@ -272,15 +276,26 @@ long int ti_build();
 typedef int (*ti_indicator_start_function)($fun_args_start);
 typedef int (*ti_indicator_function)($fun_args);
 
+
+struct ti_stream; typedef struct ti_stream ti_stream;
+typedef int (*ti_indicator_stream_new)($fun_stream_new_args);
```

Comment:
why not to return the pointer just like malloc does? we would still be able to check for errors, wouldn't we?

Reply:
We can only have one return value, so it's either:

```
int stream_new(TI_REAL const *options, ti_stream **stream);
```

or 

```
ti_stream *stream_new(TI_REAL const *options, int *return_code);
```

I don't think either is better or worse, really. The normal functions return `int`, so I just did that for consistency.

Reply:
I think returning "ti_stream" would have been a better choice, since that's the actual return type and that is what the client needs to pass again and again to compute streaming indicator. Its req. in get_info/ progress/ stream_run / stream_free.